tests/Benchmarks/System.Text.JsonLab/JsonParserPerf.cs

```diff
@@ -1,347 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using BenchmarkDotNet.Attributes;
-using Benchmarks;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
-using System.Collections.Generic;
-using System.IO;
-using System.Text.Utf8;
-
-namespace System.Text.JsonLab.Benchmarks
-{
-    // Since there are 120 tests here (8 * 15), setting low values for the warmupCount and targetCount
-    [SimpleJob(-1, 3, 5)]
-    [MemoryDiagnoser]
-    public class JsonParserPerf
```

Comment:
I believe most of the relevant benchmarks from here got moved to dotnet/performance.

If we missed some interesting tests, we can always look back at the history and grab some :)

cc @Jozkee

Reply:
Yup. Looks like it got moved to https://github.com/dotnet/performance/blob/master/src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonReader/Perf.Reader.cs.